Kit/constants.swift

```diff
@@ -22,7 +22,7 @@ public struct Popup_c_s {
 }
 
 public struct Settings_c_s {
-    public let width: CGFloat = 540
+    public let width: CGFloat = 684
```

Comment:
why did you change the settings window?

Reply:
Widening the Timezone column made the Name column unacceptably small, as the table is sized to fit the settings window. Increasing the overall width fixed that. But perhaps there is a better approach?

Reply:
if I will increase the size of the settings window for such a small feature the width will be 10000+.
- scrollable table
- resizing another tabs
- etc

but the best solution will be just crop the text inside the select with `...`

Reply:
Whatever you think best. Just be aware that timezones are most significant on the right hand side, so truncating may make them ambiguous. Maybe a text field with autocomplete (and validation) for entry would be better, then you can truncate to the left when displaying.